pkg/image/api/validation/validation.go

```diff
@@ -10,13 +11,21 @@ import (
 	"github.com/openshift/origin/pkg/image/api"
 )
 
+func MinimalNameValidation(name string, prefix bool) (bool, string) {
```

Comment:
Sort of.  All the name validators need to at least limit those characters.  The others I know about are more restrictive.